### DISCUSSION TITLE
hiding display of prompt file to start generation (-f option)

### DISCUSSION THREAD

**Discussion by l0d0v1c** [ggml-org/llama.cpp#1498] [Q&A]
Is there a way to avoid the display of a long prompt loaded with -f option without modifying the source code?

Thanks

**Comment by DannyDaemonic** [comment]
That's one of the first things I noticed. There isn't currently a way.

Edit: My work around doesn't work either. ☹️

**Reply by l0d0v1c** [reply]
I tried but the prompt is displayed anyway

**Reply by DannyDaemonic** [reply]
Sorry. I guess there's not a workaround either.

**Comment by kisonecat** [comment]
I would find this useful as well.
